src/DotNet.Status.Web.Tests/DotNet.Status.Web.Tests.csproj

```diff
@@ -4,6 +4,12 @@
     <IsPackable>false</IsPackable>
     <SignAssembly>false</SignAssembly>
   </PropertyGroup>
+  <ItemGroup>
```

Comment:
I think that this ItemGroup can be removed. Probably VS added it when deleting file in solution explorer or something.